### PR TITLE
Added Effect 'Spawn Space Ranger'

### PR DESCRIPTION
**Pull request by Last0xygen**
Fix https://github.com/gta-chaos-mod/ChaosModV/issues/490

### CODE REVIEW

File: ChaosMod/Effects/db/Peds/PedsSpawnSpaceRanger.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+    static Hash bikerHash = GET_HASH_KEY("u_m_y_rsranger_01");
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Peds/PedsSpawnSpaceRanger.cpp:9]: Think there's a leftover here :p

File: ConfigApp/Effects.cs [hunk 2]
```diff
@@ -582,6 +583,7 @@ public enum EffectType
             {EffectType.EFFECT_VEHS_CRUISE_CONTROL, new EffectInfo("Cruise Control", EffectCategory.VEHICLE, "vehs_cruise_control", true, true)},
             {EffectType.EFFECT_PLAYER_AIMBOT, new EffectInfo("Aimbot", EffectCategory.PLAYER, "player_aimbot", true)},
             {EffectType.EFFECT_PEDS_SPAWN_BIKER, new EffectInfo("Spawn Biker", EffectCategory.PEDS, "peds_spawn_biker")},
+            {EffectType.EFFECT_PEDS_SPAWN_SPACE_RANGER, new EffectInfo("Spawn Space Ranger", EffectCategory.PEDS, "peds_spawn_space_ranger", false)},
```
Comment (pongo1231) [inline ConfigApp/Effects.cs:586]: No need for the `false`

**Review by pongo1231 — CHANGES_REQUESTED** [review]

File: ChaosMod/Effects/db/Peds/PedsSpawnSpaceRanger.cpp (hunk 1 above)
Comment (Last0xygen) [reply ChaosMod/Effects/db/Peds/PedsSpawnSpaceRanger.cpp]: Maybe he’s an undercover biker, who knows... ;P 
Yeah, damn copy paste mistakes...

**Comment by pongo1231** [comment]
He should probably rather use an Unholy Hellbringer instead of the Widowmaker since otherwise he essentially kills the player instantly

**Comment by ProfessorBiddle** [comment]
> He should probably rather use an Unholy Hellbringer instead of the Widowmaker since otherwise he essentially kills the player instantly

Really bad accuracy might fix that as well

**Comment by Last0xygen** [comment]
Changed to „Unholy Hellbringer“ (untested)

**Pull request merged**